libproto/src/protos/response.proto

```diff
@@ -34,6 +34,8 @@ message Response {
         bytes transaction_proof = 20;
         // cita_getMetaData
         string meta_data = 21;
+        // eth_getBalance
+        bytes balance = 22;
```

Comment:
The balance should be uint64 type.

Reply:
If we can ensure `uint64` is enough for total balance with smallest unit, then we should use `uint64`, it's more clearly.

Now bytes is for `U256`

Reply:
@EighteenZi Can we do some math for the `balance` thing?

Reply:
yes, i used u64 first.
But the type of 'balance' in 'Account ' is U256.
If it is casted into u64, there may be something unsafe.
2 ^ 64 = 18446744073709551616 = 1.8446744073709551616e+19
1 Ether = 10^{18} Wei
So in Ethereum, it MUST be U256.

In our applyment, I can't decide which to select, u64 or u256.

Needs more discussion and dicision.

Reply:
ok, as you say.

No problem.